app/models/post.rb

```diff
@@ -0,0 +1,4 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+end
```

Comment:
Very nice work adding the associations. However, kindly add validations.

- [ ]  Validate the body and the title